test/example.test.js

```diff
@@ -1,18 +1,22 @@
 // IMPORT MODULES under test here:
 // import { example } from '../example.js';
 
+import { renderParticipants } from '../render-utils.js';
+
 const test = QUnit.test;
 
-test('time to test a function', (expect) => {
+test('renderParticipants should return a <div> with participant name', (expect) => {
     //Arrange
     // Set up your arguments and expectations
-    const expected = true;
-    
-    //Act 
+    const expected = `<p class="participant;">Kimber</p>`;
+
+    //Act
     // Call the function you're testing and set the result to a const
-    const actual = true;
+    const actual = renderParticipants({
+        name: 'Kimber',
+    });
 
     //Expect
     // Make assertions about what is expected versus the actual result
-    expect.equal(actual, expected);
+    expect.equal(actual.outerHTML, expected);
 });
```

Comment:
Awesome job testing your PURE renderParticipants function! This will be a great skill to continue practicing for future modules :)